src/test/resources/reference.conf

```diff
@@ -70,7 +70,7 @@ duos {
 # these are the settings currently used at runtime; copy them here
 # so we're testing under similar conditions.
 spray.can.host-connector {
-  max-connections = 8
+  max-connections = 4
   max-retries = 5
-  pipelining = true
+  pipelining = off
```

Comment:
these changes are not associated with this ticket, but they fix the flaky HttpOntologyDAO concurrency unit test. We made changes a while back to the runtime conf, but didn't keep the reference.conf in sync.

I don't know why unit tests pass locally (for me) but fail in Travis - either way, this seems like a decent change to make.